core/extraction/towns_fund_round_three.py

```diff
@@ -549,7 +549,7 @@ def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict) -> pd.Dat
 
     # TODO: Check we should drop rows with no source name. Or should we use another rule?
```

Comment:
we could probably also remove this TODO now, I feel like this has been addressed?